js/cart.js

```diff
@@ -1,66 +1,113 @@
-//Faie une boucle qui recupere les strings dans localStorage et les transforme en objet
-    var products = [];
-    for (var i = 0; i < localStorage.length; i++) {
-        var product = localStorage.key(i);
-        products.push(JSON.parse(localStorage.getItem(product)));
+fetch(apiUrl)
+  .then(function (res) {
+      if (res.ok) {
+      return res.json();
+      }
+  })
+
+  .then(function (teddys) {
+    //Je récupere le tableau "cart" stocké dans le localStorage
+    var cart = JSON.parse(localStorage.getItem('cart'));
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/nitatemic/AlexandreDeLemeny-Makedone_5_18062021_FRONT/pullRequest?prid=7892203)
```suggestion
    var cart = JSON.parse(localStorage.getItem("cart"));
```